storagedriver/factory/factory.go

```diff
@@ -58,7 +58,12 @@ func Create(name string, parameters map[string]interface{}) (storagedriver.Stora
 		// }
 		// return driverClient, nil
 	}
-	return driverFactory.Create(parameters)
+	d, err := driverFactory.Create(parameters)
```

Comment:
> This can probably be done in factory.Register

“can” or “should”? Based on that we can move it. I think the creation is the correct place to do this. If we can force initializations of any storage driver to be from only one place, it will help a great deal. (a.k.a factory pattern)

If we do it in factory.Register, there is a possibility that a driver is created, not registered to factory but used anyway somewhere else. So I'm more fan of doing this in Create.